Sources/Components/Profile/ReputationView.swift

```diff
@@ -325,7 +325,7 @@ extension ReputationView {
 
         let imageView = UIImageView(image: model.category.icon)
         imageView.translatesAutoresizingMaskIntoConstraints = false
-        imageView.tintColor = .stone
+        imageView.tintColor = .textSecondary
```

Comment:
Is this correct? A bit confusing to assign a text color to an image view

Reply:
in this case @ninarg you can make a private extension of UIColor to assign a name to this semantic color and return `.textSecondary` as value, as the color may be correct, but not the semantics of it